frontstage/views/secure_messaging/create_message.py

```diff
@@ -50,9 +53,13 @@ def send_message(party_id, is_draft):
     subject = form['subject'].data if form['subject'].data else form['hidden_subject'].data
     message_json = {
         'msg_from': party_id,
+        'msg_to': ['GROUP'],
```

Comment:
Are you aware this currently breaks response operations messages view? As the msg_to field does not get populated when retrieving a message.

Reply:
Not following how it will break it ?

Reply:
If a respondent sends a message to internal user, when an internal user tries to go to the messages list view they will be presented with a 500 internal server error. This is because the msg_to field comes back as an empty list. Looks like the reason for this is because internal_user_service.get_user_details is not implemented yet.

Reply:
Never mind looks like there's a PR already to fix this https://github.com/ONSdigital/ras-secure-message/pull/156

Reply:
There is a change imminent which will implement that , the original implementation is flawed , it should have returned dummy data . We still have to handle the case of the msg_to being group when we retrieve data .

Reply:
I think this is actually a bug in r-ops, I've raised it with out team. It happens with messages sent from the current master branch.